package.json

```diff
@@ -52,10 +50,16 @@
     "style-loader": "^3.3.1",
     "vue-loader": "^15",
     "vue-template-compiler": "^2.7.14",
-    "vuex": "^3.6.2",
     "webpack": "^5.75.0",
     "webpack-cli": "^5.0.0"
   },
+  "peerDependencies": {
+    "tiptap": "^1.32.2",
+    "tiptap-commands": "^1.17.0",
+    "tiptap-extensions": "^1.35.2",
+    "vue": "^2.7.14",
+    "react": "16.14.0"
```

Comment:
why do we depend on react here?

Reply:
because of storybook.

Reply:
ok but storybook comes with its own package.json where it defines its dependencies... and as demosplan-ui does not require react directly, i do not understand why we have to define it as a peer dependency here.

Reply:
maybe we don't need to. But after adding it, all the warnings disappeared